plugins/TodayAtMun/DiaryUtil.py

```diff
@@ -0,0 +1,105 @@
+from datetime import datetime, timedelta
+
+
+class DiaryUtil:
+    """Provides methods to manipulate dates and lookup/match parsed data."""
+
+    def __init__(self, diary: dict[str, str]):
+        self.diary = diary
+        self.date = DiaryUtil.get_current_date()
+
+    @staticmethod
+    def get_current_date() -> datetime:
+        return datetime.now()
+
+    @staticmethod
+    def str_to_datetime(str_date: str):
```

Comment:
i assume you want a type here? just noticed this lacks a typing but most other things have them